src/lib/fragments/entryProps.ts

```diff
@@ -0,0 +1,51 @@
+import gql from "graphql-tag";
+export default gql`
+  fragment EntryProps on Entry {
+    __typename
+    ... on PageMetadata {
+      sys {
+        id
+      }
+    }
+
+    ... on ImageWrapper {
+      sys {
+        id
+      }
+      __typename
+      internalTitle
+      altText
+      linkedImage {
+        title
+        description
+        contentType
+        fileName
+        size
+        url
+        width
+        height
+      }
+      imageCategory {
+        categoryName
+        categoryDescription
+      }
+    }
+    ... on Contact {
+      sys {
+        id
+      }
+      __typename
+      entityName
+      phoneExt
+      email
+      # location {
+      #   name
+      #   streetAddress1
+      #   streetAddress2
+      #   city
+      #   state
+      #   zip
+      # }
```

Comment:
strangely, this bit was working fine in the Contentful GraphiQL IDE, but I was getting this error in the GraphQL validation in local development: `Unable to find field "location" on type "Contact"!`

Reply:
Do you need to do `npm run gen-schema` maybe?

Reply:
hmmm that's giving its own problems:
`Query cannot be executed. Requested environment does not exist in the space`. Am I missing something being up-to-date?

I thought all the schema updating happened in other scripts run before `dev`. Strange that this one issue is occurring.